editor/writing-flow/index.js

```diff
@@ -94,4 +108,18 @@ class WritingFlow extends Component {
 	}
 }
 
-export default WritingFlow;
+export default connect(
+	( state ) => {
+		return {
+			insertionPoint: getBlockInsertionPoint( state, false ),
```

Comment:
To the issue of adding the specific, `defaultAtEnd` parameter, maybe we should not use the `getBlockInsertionPoint` selector and use `getSelectedBlockIndex` (doesn't exist yet) instead